app/views/components/gds/tariffExpander.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import config.AppConfig
+@import forms.DeclarationPage
+@import models.DeclarationType.DeclarationType
+
+@this(govukDetails : GovukDetails)
+
+@(page: DeclarationPage, journey: DeclarationType)(implicit messages: Messages, appConfig: AppConfig)
+
+@htmlElements = @{ page.defineTariffContentKeys(journey).map( key =>
+ messages(s"$key.text", components.gds.tariffLink(key))
+).mkString("</br></br>")}
+
+@govukDetails(Details(
+ attributes = Map("id" -> "tariffReference"),
+ summary = Text(messages("site.details.summary_text")),
+ content = HtmlContent(htmlElements)
+))
```

Comment:
Just a cosmetic one... Increasing the indentation of lines 27 and 31-33 of at least one blank?